tests/ballerina-unit-test/src/test/resources/test-src/types/table/table_type.bal

```diff
@@ -137,17 +137,13 @@ function testToJson() returns (json) {
         password: "",
         poolOptions: { maximumPoolSize: 1 }
     };
-
-    try {
-        table dt = check testDB->select("SELECT int_type, long_type, float_type, double_type,
+    table dt = check testDB->select("SELECT int_type, long_type, float_type, double_type,
                   boolean_type, string_type from DataTable WHERE row_id = 1", ());
-        json result = check <json>dt;
-        // Converting to string to make sure the json is built before returning.
-        _ = result.toString();
-        return result;
-    } finally {
-        testDB.stop();
-    }
+    json result = check <json>dt;
+    // Converting to string to make sure the json is built before returning.
+    _ = result.toString();
+    testDB.stop();
```

Comment:
We need to make sure the database endpoint stop execution event in the error scenarios. In here we are using check in line no 142. So in error situation testDB.stop will not call. Please modify other places as well.